gravitee-am-test/api/commands/management/application-management-commands.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {getDomainManagerUrl} from "./utils";
+
+const request = require('supertest');
+
+export const createApplication = (domainId, accessToken, body, status: number = 201) =>
+    request(getApplicationUrl(domainId, null))
```

Comment:
Annoyingly, it seems you have to call the http method function (get, post etc) directly after `request` which makes it unfeasible to have a common `request` used by all these other functions (as was the case with Cypress).